deployment/docker-swarm/hub/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.6'
 services:
   alertdb:
-    image: blackducksoftware/alert-database:1.0.2
+    image: blackducksoftware/blackduck-alert-db:ALERT_VERSION_TOKEN
```

Comment:
We will want a release note that alert-database has been renamed blackduck-alert-db with a new version number.  It's only a change in the naming of the image and no changes to functionality.